BE/Api.Access.Layer/Controllers/UserController.cs

```diff
@@ -50,11 +49,28 @@ public async Task<IActionResult> UpdateUser([FromBody] BusinessUserModel model)
 
         [ProducesResponseType((int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
-        [HttpGet("read")]
-        public async Task<IActionResult> GetUserByEmail([FromBody] string email)
+        [HttpGet("GetUserByEmail")]
```

Comment:
@BoskoGogic95 See if these changes of GetUserByEmail method are okay

Reply:
Yea it's okay bro. Thanks for notify about that changes.